chrome-os-best-practices/lib/main.dart

```diff
@@ -237,12 +239,12 @@ class ChatMessage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return SizeTransition(
-    sizeFactor: CurvedAnimation(
+      sizeFactor: CurvedAnimation(
         parent: animationController,
-        curve: Curves.easeOut
-    ),
-    axisAlignment: 0.0,
-    child: Container(
+        curve: Curves.easeOut,
+      ),
+      axisAlignment: 0.0,
+      child: Container(
```

Comment:
Since you're in here, there are two more instances of <Widget>[] that don't need the type specifier.